piccolo_api/crud/endpoints.py

```diff
@@ -820,19 +828,31 @@ async def post_single(
             return Response("Unable to save the resource.", status_code=500)
 
     @apply_validators
-    async def delete_all(
+    async def delete_bulk(
         self, request: Request, params: t.Optional[t.Dict[str, t.Any]] = None
     ) -> Response:
         """
-        Deletes all rows - query parameters are used for filtering.
+        Bulk deletes rows - query parameters are used for filtering.
         """
         params = self._clean_data(params) if params else {}
         split_params = self._split_params(params)
+        split_params_ids = split_params.ids.split(",")
 
         try:
-            query = self._apply_filters(
-                self.table.delete(force=True), split_params
-            )
+            query: t.Union[
+                Select, Count, Objects, Delete
+            ] = self.table.delete()
```

Comment:
Should this just be `query: Delete = self.table.delete()`

Reply:
Unfortunately that raise `mypy` error (`error: Incompatible types in assignment (expression has type "Union[Select, Count, Objects, Delete]", variable has type "Delete"`)
Because of that I added `t.Union[Select, Count, Objects, Delete]` type annotation.